environment.yml

```diff
@@ -1,7 +1,6 @@
 channels:
   - usgs-astrogeology
   - conda-forge
-  - probcomp
```

Comment:
This may need to be added back in however. Getting an error in jenkins that this channel cannot be accessed